tests/koan/test_utils.py

```diff
@@ -0,0 +1,19 @@
+from unittest.mock import MagicMock
+
+import distro
+import pytest
+
+from koan import utils
+
+
+@pytest.mark.parametrize("test_input,expected",
+                         [("suse", "suse"), ("rhel", "redhat"), ("centos", "centos"), ("notexisting", "unkown")])
```

Comment:
I think there is a typo :point_right: "unkown"

Reply:
Should be fixed now